scripts/release.js

```diff
@@ -0,0 +1,4 @@
+var execSync = require('child_process').execSync;
+
+execSync("ember build --environment production");
+execSync("npm publish");
```

Comment:
This actually seems a bit scary to auto-trigger. I feel like we just need:
`execSync("ember build --environment production");`

And that it should run as part of the `prepublish` hook.